frontend/src/components/DialogAddNewGoal.tsx

```diff
@@ -11,12 +11,12 @@ type Dialog_AddNewGoalProps = {
     addGoal: (newGoal: Goal) => void;
 }
 
-export default function DialogAddNewGoal(props: Readonly<Dialog_AddNewGoalProps>){
+export default function DialogAddNewGoal(props: Readonly<Dialog_AddNewGoalProps>) {
     const [goals, setGoals] = useState<Goal[]>(props.user?.goals || []);
     const [goalName, setGoalName] = useState("");
     const [goalPrice, setGoalPrice] = useState(0);
```

Comment:
Das hier könnte man evtl auch in einen State packen - muss man aber nicht, nur ein Vorschlag. Also z.B.
```
type Goal = {
  name: string,
  price: int
}
const [goal, setGoal] = useState<Goal>()
```